### DISCUSSION TITLE
Why is there a specification that allows me to choose the Faraday version?

### DISCUSSION THREAD

**Discussion by ytkg** [octokit/octokit.rb#1624] [Q&A]
Why is there a specification that allows me to choose the Faraday version?

**Comment by timrogers** [comment]
Can you explain your question a little more? I'm not sure that I understand.

**Reply by ytkg** [reply]
I read the following code and wondered about supporting faraday v1.
https://github.com/octokit/octokit.rb/blob/de112884bb501a1823c5b73e1a4a09956760d017/Gemfile#L36-L43

Are there any plans to support Faraday v2 only?

**Reply by ytkg** [reply]
Sorry if my English is weird.
I use a translation tool.
